src/mindMapPreviewWebview.ts

```diff
@@ -1,5 +1,11 @@
 import * as vscode from 'vscode';
 import * as path from "path";
+import * as fs from "fs";
+// const html2svg = require('html2svg');

```

Comment:
Debug comments like this should not appear in the master branch. Please delete it. Thanks.